apps/client/src/app/model/character/lostark-class.ts

```diff
@@ -1,34 +1,31 @@
 export enum LostarkClass {
-  DESTROYER,
-  UNRELEASED,
-  ARCANA,
-  BERSERKER,
-  WARDANCER,
-  DEADEYE,
-  UNRELEASED0,
-  GUNLANCER,
-  GUNNER,
-  SCRAPPER,
-  MAGE,
-  SUMMONER,
-  WARRIOR,
-  SOULFIST,
-  SHARPSHOOTER,
-  ARTILLERIST,
-  BARD,
-  GLAIVIER,
-  ASSASSIN,
-  DEATHBLADE,
-  SHADOWHUNTER,
-  PALADIN,
-  SCOUTER,
-  REAPER,
-  UNRELEASED4,
-  GUNSLINGER,
-  UNRELEASED5,
-  STRIKER,
-  SORCERESS,
-  ARTIST
+  DESTROYER = 0,
+  SLAYER = 1,
+  ARCANA = 2,
+  BERSERKER = 3,
+  WARDANCER = 4,
+  DEADEYE = 5,
+  GUNLANCER = 7,
+  GUNNER = 8,
+  SCRAPPER = 9,
+  MAGE = 10,
+  SUMMONER = 11,
+  WARRIOR = 12,
+  SOULFIST = 13,
+  SHARPSHOOTER = 14,
+  ARTILLERIST = 15,
+  BARD = 16,
+  GLAIVIER = 17,
+  ASSASSIN = 18,
+  DEATHBLADE = 19,
+  SHADOWHUNTER = 20,
+  PALADIN = 21,
+  SCOUTER = 22,
+  REAPER = 23,
+  GUNSLINGER = 25,
+  STRIKER = 27,
+  SORCERESS = 28,
+  ARTIST = 29,
```

Comment:
I'm fine with this change but fwiw, this doesn't change anything 😄, TS enums have an implicit index which is why the UNRELEASEDX values were in the original enum.

Reply:
Yep! Did it to remove the UNRELEASEDX label, IMO it looked more organized this way.